### PR TITLE
50 calendar component

### PR DESCRIPTION
**Pull request by bochka123**

### CODE REVIEW

**Comment by bochka123** [comment]
image

File: frontend/src/app/shared/shared.module.ts [hunk 1]
```diff
@@ -30,6 +35,10 @@ import { YoutubePlayerComponent } from './components/youtube-player/youtube-play
         NgSelectModule,
         MatProgressSpinnerModule,
         MatSnackBarModule,
+        MatButtonModule,
+        MatDatepickerModule,
+        MatNativeDateModule,
+        MatInputModule,
```
Comment (tatianahutii) [inline frontend/src/app/shared/shared.module.ts:41]: These items should be a part of the Material module

Comment (tatianahutii) [reply frontend/src/app/shared/shared.module.ts:41]: Please move all mat items from this file to the Material items

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Review by Anatolii-Ivanov — COMMENTED** [review]
1. Can you create another migration and check that no changes will be applied? Especially that BirthDate is static
2. Update calendar in userProfile/details page too, please

**Comment by bochka123** [comment]
image

**Comment by bochka123** [comment]
image

File: backend/EasySpeak.Core/EasySpeak.Core.DAL/Migrations/20230326214449_ChangeAgeToDateBirth.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                keyValue: 2L,
+                columns: new[] { "BirthDate", "Email", "FirstName", "ImagePath", "LanguageLevel", "LastName", "Sex", "Status" },
+                values: new object[] { new DateTime(2003, 6, 5, 0, 0, 0, 0, DateTimeKind.Unspecified), "Jon.Abshire@gmail.com", "Jon", "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/949.jpg", 1, "Abshire", 1, 3 });
+
+            migrationBuilder.UpdateData(
+                table: "Users",
+                keyColumn: "Id",
+                keyValue: 3L,
+                columns: new[] { "BirthDate", "Email", "FirstName", "ImagePath", "IsBanned", "LanguageLevel", "LastName", "Sex", "Status" },
+                values: new object[] { new DateTime(2003, 6, 5, 0, 0, 0, 0, DateTimeKind.Unspecified), "Kurt.Gulgowski93@gmail.com", "Kurt", "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/1117.jpg", false, 3, "Gulgowski", 0, 3 });
```
Comment (Anatolii-Ivanov) [inline backend/EasySpeak.Core/EasySpeak.Core.DAL/Migrations/20230326214449_ChangeAgeToDateBirth.cs:42]: Why you update the whole user here...

File: backend/EasySpeak.Core/EasySpeak.Core.DAL/EasySpeak.Core.DAL.csproj [hunk 3]
```diff
@@ -20,10 +20,6 @@
   <ItemGroup>
     <ProjectReference Include="..\EasySpeak.Core.Common\EasySpeak.Core.Common.csproj" />
   </ItemGroup>
-  
-  <ItemGroup>
-    <Folder Include="Migrations\" />
-  </ItemGroup>
```
Comment (Anatolii-Ivanov) [inline backend/EasySpeak.Core/EasySpeak.Core.DAL/EasySpeak.Core.DAL.csproj:23]: Are you sure?

File: backend/EasySpeak.Core/EasySpeak.Core.DAL/Migrations/20230326214449_ChangeAgeToDateBirth.cs (hunk 2 above)
Comment (bochka123) [reply backend/EasySpeak.Core/EasySpeak.Core.DAL/Migrations/20230326214449_ChangeAgeToDateBirth.cs:42]: In the next migrations everything is ok

File: backend/EasySpeak.Core/EasySpeak.Core.DAL/EasySpeak.Core.DAL.csproj (hunk 3 above)
Comment (bochka123) [reply backend/EasySpeak.Core/EasySpeak.Core.DAL/EasySpeak.Core.DAL.csproj]: Actually, I didn`t remove it deleted itself, so I got it back)

**Review by SidletskyiOlexandr — APPROVED** [review]

**Pull request merged**